documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -582,6 +582,28 @@ Map<K, V> getGroup(String groupName);
 void removeGroup(String groupName);
 ----
 
+=== Scattered Mode
+
+Scattered mode is very similar to Distribution Mode as it allows linear scaling of the cluster.
+It allows single node failure by maintaining two copies of the data (as Distribution Mode with numOwners=2).
+Unlike Distributed, the location of data is not fixed; while we use the same Consistent Hash algorithm
+to locate the primary copy, the backup copy is on the node that wrote the data last time (or another node
+when the data was last written on primary owner).
+
+This has the advantage of single RPC for any write (Distribution Mode requires one or two RPCs), but reads
+have to always hit the primary owner. That results in faster writes, and slower reads, and therefore this
+mode is more suitable for write-intensive applications.
+
+Storing multiple backup copies also results in slightly higher memory consumption. In order to remove out-of-date
+backup copies, invalidation messages are broadcast in the cluster, which generates some overhead. This makes
+scattered mode less performant in very big clusters (this behaviour might be optimized in the future).
+
+When a node crashes, the primary copy may be lost. Therefore, the cluster has to reconcile the backups
+and find out the last written backup copy. This process results in more network traffic during state transfer.
+
+Currently it is not possible to use scattered mode in transactional cache. Asynchronous replication
```

Comment:
Yes, this is checked during the validation phase of programmatic configuration, and transactions are not in schema for scattered cache.